src/main/java/app/view/mapBuilder/SettingsPane.java

```diff
@@ -1,17 +1,51 @@
 package app.view.mapBuilder;
 
+import app.controller.settings.Settings;
+import javafx.geometry.Insets;
 import javafx.scene.control.Label;
 import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
 
 public class SettingsPane extends StackPane // Might need to change this to extend something else
 {
     private StartMenu startMenu;
+    private Settings s;
 
-    public SettingsPane(StartMenu startMenu)
+    public SettingsPane(StartMenu startMenu, Settings s)
     {
         this.startMenu = startMenu;
+        this.s = s;
 
-        Label label = new Label("Settings Go Here");
-        this.getChildren().add(label);
+        // Labels displaying current settings information.
+        VBox vbox = new VBox(10);
+        Label header = new Label("Current simulation settings:");
+        header.setFont(Font.font("", FontWeight.BOLD, 24));
+        Label mapName = new Label("Map Name - " + s.getName());
+        mapName.setFont(new Font(16));
+        Label gameMode = new Label("Game Mode - " + s.getGameMode());
```

Comment:
Can you render all of these labels with the info in boxes, so that they look like:

Game Mode: [ 12 ]

Where the 12 is an editable text area?

Reply:
Done, Any other changes let me know.